src/com/intellij/idea/plugin/hybris/tools/logging/actions/LoggerActions.kt

```diff
@@ -57,15 +56,15 @@ abstract class AbstractLoggerAction(private val logLevel: String, val icon: Icon
                             AbstractHybrisHacHttpClient.DEFAULT_HAC_TIMEOUT
                         )
 
-                        val serverName = RemoteConnectionUtil.getActiveRemoteConnectionSettings(project, RemoteConnectionType.Hybris).toString()
+                        val server = RemoteConnectionUtil.getActiveRemoteConnectionSettings(project, RemoteConnectionType.Hybris)
 
                         Notifications.create(
                             if (result.statusCode == 200) NotificationType.INFORMATION else NotificationType.ERROR,
```

Comment:
this is only code sugar, let's include some other improvements, like `result.statusCode == 200` -> `val isSuccess`, etc.

Reply:
fixed